CMakeLists.txt

```diff
@@ -65,8 +65,11 @@ include(CMakeDependentOption)
 option(BUILD_DAEMON "Build bitcoind executable." ON)
 option(BUILD_GUI "Build bitcoin-qt executable." OFF)
 option(BUILD_CLI "Build bitcoin-cli executable." ON)
```

Comment:
`bitcoin-cli` is also referenced from the functional tests. Shouldn't it also be treated like the others, e.g. `bitcoin-tx`?

Reply:
I'd keep it `ON` by default and, [as you said](https://github.com/hebasto/bitcoin/pull/196#discussion_r1605696997),  independently from tests.